web/rest/client/features/provider/conditionalRoutesCondition/putConditionalRoutesCondition.feature

```diff
@@ -15,6 +15,7 @@ Feature: Update conditional routes conditions
           "routeType": "ivr",
           "numberValue": null,
           "friendValue": "",
+          "id": 1,
```

Comment:
you cannot put an element with an id in the body, you are currently updating /conditional_routes_conditions/1